### PR TITLE
PP-1059: added one more regression scenario

### PR DESCRIPTION
**Pull request by anamikau**
#### Issue-ID

* **[PP-1059](https://pbspro.atlassian.net/browse/PP-1059)**

#### Problem description
* added one more regression scenario

#### Cause / Analysis
* added one more regression scenario

#### Solution description
* added one more regression scenario

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [ ] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
- [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
- [ ] I have added  **manual test(s) to the Jira ticket and explained why PTL is not appropriate** for this case.
- [ ] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [x] I have attached **test logs to the Jira ticket** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

File: test/tests/functional/pbs_equiv_classes.py [hunk 1]
```diff
@@ -1878,3 +1878,53 @@ def test_queue_resav(self):
...
+        self.server.expect(RESV, a, id=rid2)
+
+        r1name = rid1.split('.')
+        r2name = rid2.split('.')
+        a = {ATTR_queue: r1name[0], 'Resource_List.select': '1:ncpus=1'}
+        j1 = Job(TEST_USER, a)
+        jid1 = self.server.submit(j1)
+        # Sleeping in between job submission to allow
+        # scheduler pick first job
+        time.sleep(1)
```
Comment (bhroam) [inline test/tests/functional/pbs_equiv_classes.py]: Instead of a sleep, do an expect()

**Comment by bhroam** [comment]
Codacy is reporting a couple of unused variables.

**Comment by anamikau** [comment]
@bhroam addressed your review comments. All checks have passed. Kindly review.

**Comment by tpai26** [comment]
Looks good to me.

File: test/tests/functional/pbs_equiv_classes.py [hunk 2]
```diff
@@ -1902,18 +1902,18 @@ def test_overlap_resv(self):
         a = {ATTR_queue: r1name[0], 'Resource_List.select': '1:ncpus=1'}
         j1 = Job(TEST_USER, a)
         jid1 = self.server.submit(j1)
-        # Sleeping in between job submission to allow
-        # scheduler pick first job
-        time.sleep(1)
+        self.server.expect(JOB, {'job_state': 'Q'}, id=jid1)
```
Comment (bhroam) [inline test/tests/functional/pbs_equiv_classes.py]: Just checking for the job in the 'Q' state is not enough to make sure the scheduler has seen the job.  The job will be in the 'Q' state when it is initially submitted.  To make sure the scheduler has seen the job and it remains in the 'Q' state, you need to check that the comment is set as well.

**Comment by anamikau** [comment]
@bhroam addressed your comments. All checks have passed. please do the needful.

File: test/tests/functional/pbs_equiv_classes.py [hunk 3]
```diff
@@ -1902,18 +1902,18 @@ def test_overlap_resv(self):
         a = {ATTR_queue: r1name[0], 'Resource_List.select': '1:ncpus=1'}
         j1 = Job(TEST_USER, a)
         jid1 = self.server.submit(j1)
-        self.server.expect(JOB, {'job_state': 'Q'}, id=jid1)
+        self.server.expect(JOB, 'comment', op=SET, id=jid1)
```
Comment (bhroam) [inline test/tests/functional/pbs_equiv_classes.py]: So I usually do both here.  I expect comment as set and then check the job in the Q state.  If you just check for the comment being set, the job could be in other states.  For example, if the job runs, the comment will be set.

**Comment by bhroam** [comment]
Please squash and rebase for the final merge.

**Comment by anamikau** [comment]
thanks @bhroam . @tpai26 can you sign off again

**Comment by anamikau** [comment]
@bhroam squashed and rebased. please take it forward once the checks have passed.

**Comment by anamikau** [comment]
All checks have passed. please take it forward.

**Comment by bhroam** [comment]
I merged in the PR.

**Pull request merged**